src/libstd/sys/cloudabi/abi/cloudabi.rs

```diff
@@ -115,6 +115,7 @@
 
 #![no_std]
 #![allow(non_camel_case_types)]
+#![allow(deprecated_in_future)] // FIXME: using `mem::uninitialized()`
```

Comment:
Are you sure this shouldn't be `allow(deprecated)`? After all that's the attribute you removed from `mod.rs`.

Reply:
Oops. That's my mistake while rebasing.